Bottom: 1b2ffa132d59866ee2c5ae798eab4b94142bfbdb
Top:    c7f603cfe3c73cbd0ed631585f5105713071ee74
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-26 09:10:46 -0500

Refresh of all-docker_volume

---
diff --git a/roles/docker-mediawiki/tasks/main.yml b/roles/docker-mediawiki/tasks/main.yml
index 69e4e55c..86587ab1 100644
--- a/roles/docker-mediawiki/tasks/main.yml
+++ b/roles/docker-mediawiki/tasks/main.yml
@@ -141,6 +141,7 @@
       WIKI_MAIL_USER: 'wiki'
       WIKI_MAIL_PASSWORD: '{{ wiki_email_password | quote }}'
     image: '{{ wiki_image_name }}:{{ docker_image_tag }}'
+    state: started
 
 # *****************************************************************************
 # restore?
